files-community/Files
# Fix: Fixed ComboBox position shift

**Resolved / Related Issues**
   Closes #14965 

**Validation**
How did you test these changes?
   1. Open Files Dev build
   2. Click settings button
   3. Go to pages where `ComboBox` -es are present (General, Appearance, Folders, Layout)
   4. Check the `ComboBox` -es to see if the changes have been applied

**Screenshots (optional)**

https://github.com/files-community/Files/assets/87632612/705f4a71-70a5-40d1-a307-52df82d6b8b0
